Common/Commands/Config.h

```diff
@@ -127,7 +127,8 @@ class ConfigResponse : public BaseResponse
 				(requiresNetwork ? 1 : 0) + 
 				(enablesCheckout ? 1 : 0) + 
 				(enablesLocking  ? 1 : 0) +
-				(enablesRevertUnchanged  ? 1 : 0);
+				(enablesRevertUnchanged  ? 1 : 0)+
+                (enablesRevertUnchangedOnSubmit ? 1 : 0);
```

Comment:
Strange indentation everywhere. Looks like previous source files used actual tabs for indentation, and yours is using spaces.